src/profile/components/ChangePassword/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,213 @@
+import { useState } from "react"
+import { useNavigate } from "react-router-dom"
+import { Controller, useFormContext } from "react-hook-form"
+import { Button, Copy, Flex, FormInput, Spacer, Stack, Hidden } from "@/shared/components"
+import { IconClarityEye } from "@/shared/icons"
+import { ChangeInput } from "@/api/types"
+import { SShowPasswordButton } from "./ChangePasswordForm.styls"
+
+export const ChangePasswordForm = ({
+  defaultValues,
+  isLoading,
+  isPasswordChanged,
+}: {
+  defaultValues: ChangeInput
+  isLoading: boolean
+  isPasswordChanged: boolean
+}) => {
+  const navigate = useNavigate()
+
+  const {
+    control,
+    register,
+    watch,
+    formState: { errors },
+  } = useFormContext<ChangeInput>()
+  const [passwordShown, setPasswordShown] = useState(false)
```

Comment:
add `const [currentPasswordShown, setCurrentPasswordShown] = useState(false)` and `const toggleCurrentPasswordVisibility = () => setCurrentPasswordShown(!currentPasswordShown)`

Reply:
Done